### ISSUE TITLE
luks tutorial is still horrible to follow

### ISSUE THREAD

**Issue by Tcll** [void-linux/void-docs#502]
This is an unresolved continuation of [my previous issue](https://github.com/void-linux/void-linux.github.io/issues/76) on this topic.
all you guys did was copy/paste the already horrible to follow tutorial from the [deprecated wiki](https://wiki.voidlinux.org/Full_Disk_Encryption_w/Encrypted_Boot) to the [new docs page](https://docs.voidlinux.org/installation/guides/fde.html)

while my previous issue applies to Void 20181111, the issue still applies to the current 20191109 release
`void-installer` still doesn't support luks, and the new docs page still makes no attempt at calling `void-installer` for custom configuration like my previous issue does.
it just automatically assumes this is what every installation (desktop, server, router, firewall, devtest) should require...
or rather it was designed as an example for a particular use case to show the Void Linux developers (not it's users) how to support luks.

why should users have to be shown a developer tutorial?
why can't the developers make `void-installer` support luks like the tutorial was clearly aimed at?

**Comment by ericonr** [comment]
>why can't the developers make void-installer support luks like the tutorial was clearly aimed at?

We are all volunteers here with a pretty high workload already, and it seems no one has been able to get to this.

Furthermore:

- `void-installer` not supporting LUKS isn't relevant for this repo.
- the chroot installation _is_ used by several users
- open PRs, not issues. Alternatively, be less demanding. Language like "still makes no attempt" doesn't help anyone
- showing a more advanced guide for FDE installation in the "Advanced Guides" section of the handbook feels very topical to me

**Comment by Tcll** [comment]
I'm not a linux developer, I'm just a user, please don't ask me to open a PR for something I'm not knowledgeable about. ;)
I'm making a suggestion regarding `void-installer`, and the only reason I currently sound demanding is because my last issue was closed without reason and was not solved.
this issue pertains to improvement requests for the void docs
I do not have the ability to make these improvements, but I can provide suggestions.

to add, I'm already busy with my own projects in python, though you won't find my projects here as I support freedom.

**Comment by hippi777** [comment]
hi there :) 

tldr: u r free to ignore the whole message at will :D

i actually work on a new installer, that takes a config instead of being an interactive eye-candy, but it still isnt ready to be published... later it could get a validator to make a sanity-check on the config, and a front-end that is more human friendly, but i think those tasks will be left to others, cuz i like the config style, and a handmade config can be more pretty and advanced than a generated one... otherwise the config is already kinda advanced, i needed tree structures (in bash... eww... imho this is the best thing that ever happened to bash :D kinda much anything can be done with them one can dream about (even arbitrary types can be assigned to the values of them as a side-effect :D ), and i made a whole lib for them, and some other libs too to make bash friendlier...) i needed them especially cuz of the flexibility that various fs layouts can require, and a good thing is that u dont need to mess any or much with the config to make ur next installation :) i even think that the current config wont need any or much change other than extending its possibilities :) i have tried to do my best to make it easy-to-understand and more-or-less educational, cuz i have read something like the 3/4 of the internet to learn all what i wanted, but u need patience to take this way, i dunno how far i will reach by the time i can dedicate to it, but i believe it wont be that hard or alien stuff to be unable to use it... i think i can spend 1-2 months more on it, so stay tuned... otherwise it still depends on the void folks if they will like it or not to get official support, as i wrote it in my style, it has some hacks here-and-there (basically enclosed into pretty functions :D ), i didnt align it to the fs hierarchy standard, while it is self-contained and portable, and there will be uncovered things that i believe wont make any change in an existing config, but the possibilities are open for them... like i dont use btrfs, zfs or raid, or any other bootloaders than grub2, but nothing prevents their advanced integration, and on the other hand, luks and lvm will be supported initially, and i have some basic good-to-know stuffs about various luks settings and disk layout stuffs, as i really did my research on these :D i just hope i can make a reasonable progress in time and i can give it an acceptable shape too by that time, while i think i could already give it nice funds... also, fs creation and system installation are two different stuffs, u can do anything with as many devices as u want and do as many system installations as u want even in a single config where u can use ur devices and partitions on different ways, so u can make a multi-boot machine or a chroot based subsystem or whatever u can think about, like any sane package managers could be integrated to support other systems too, but im not sure if i really want that or not (i dunno if i will lead the project or let the community do that, or how much actual pain could it bring...), but the possibility is open even for this :D actually i want it to be the representation of a fully baked system as much as possible... the most important question is where would it be the best to announce it when it will be ready :) (maybe reddit and the repo of the void installer) i wanted to make the fundamental stuffs on my own, but i dunno if teamwork would help me more or hold me up more, but at least now im free to mess with anything i would like to, and there is nothing now like sane isolated self contained commits (or version management, or versions... until reaching something that could called to be 1.0), so i can polish a lotsa different things at once, and rollback is for those who dont know what tdd is or what they do/want X'D

**Comment by Tcll** [comment]
@hippi777 while I'm not a fan of the config style, I'm not against it :)
I like having the option, though I'd prefer an easy and well-coded interactive solution.

I'm not using Void to experiment with building my own Linux distro
I'm using Void cause it's the only distro I know of that's actually security focused
and I help others install it as well so they can have that security.
I'd like to see it improve to make their lives easier.

but hey, it's good to know a new installer is in the works :)

**Comment by Tcll** [comment]
ok @zdykstra @sgn could you please explain what part of my post you're downvoting.
is it the fact I'm making suggestions instead of doing actual work?
or is it the fact Microsoft offers the illusion of freedom?

**Comment by sgn** [comment]
Dear Sir/Madame,

We're so sorry to hear your bad experience with our installer.

Please accept this message as our acknowledge that we've received your complaint.

We'll hand this issue over to the Customer Support Specialist.

In the meantime, in order to speed up the support, would you mind provide us your Privilege Customer Number!
We'll get a Business Analysis Specialist to work with you in order to understand your requirement better and we'll take further action from there.

Your sincerely,

**Comment by avb85** [comment]
Yeah, it's been a year since then. And now, 2021, autumn. And the problem is still not solved.
Developers, please fix the installation manual for full-disk encryption. The manual located on the official website should be accurate, without errors.

**Comment by zdykstra** [comment]
> Yeah, it's been a year since then. And now, 2021, autumn. And the problem is still not solved.
> Developers, please fix the installation manual for full-disk encryption. The manual located on the official website should be accurate, without errors.

What, specifically, is wrong with the documentation for full-disk encryption? If you've recently run through the documentation and have noticed problems (typos, incorrect/incomplete steps, etc), please note them in detail.

**Comment by Vaelatern** [comment]
This issue will now be closed.

While I appreciate the concerns people have with the more advanced installation guides, this project works best if all involved have a "how can I help" attitude. This issue has not inspired this attitude.

When opening followups, please either open pull requests with proposed fixes, or open specific issues  such as "this paragraph is unclear, is it referring to fizzes or buzzes?" so a volunteer can come along and reasonably correct any mistakes.
